packages/texinfo.rb

```diff
@@ -3,30 +3,31 @@
 class Texinfo < Package
   description 'Texinfo is the official documentation format of the GNU project.'
   homepage 'https://www.gnu.org/software/texinfo/'
-  version '6.8-1'
+  version '7.0'
   license 'GPL-3'
   compatibility 'all'
-  source_url 'https://ftpmirror.gnu.org/texinfo/texinfo-6.8.tar.xz'
-  source_sha256 '8eb753ed28bca21f8f56c1a180362aed789229bd62fff58bf8368e9beb59fec4'
+  source_url 'http://ftpmirror.gnu.org/texinfo/texinfo-7.0.tar.xz'
```

Comment:
Why was `https` changed to `http`?

Reply:
I copied the URL from the release announcement.

Fixed.